From 8953174da9c4199a63c94832571a1fb77aac04ee Mon Sep 17 00:00:00 2001
From: Peter Zhigalov <peter.zhigalov@gmail.com>
Date: Sat, 2 May 2026 14:09:01 +0700
Subject: [PATCH 2/2] Fix ArgbConstants declarations

---
 source/convert.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source/convert.cc b/source/convert.cc
index d9fb5477..666360ae 100644
--- a/source/convert.cc
+++ b/source/convert.cc
@@ -22,9 +22,9 @@
 
 #ifdef __cplusplus
 namespace libyuv {
+extern "C" {
 extern const struct ArgbConstants kArgbI601Constants;
 extern const struct ArgbConstants kArgbJPEGConstants;
-extern "C" {
 #endif
 
 // Subsample amount uses a shift.
-- 
2.47.3

