diff -Nur lcms2-2.19_orig/src/cmslut.c lcms2-2.19/src/cmslut.c
--- lcms2-2.19_orig/src/cmslut.c	2026-05-02 13:48:57.835308172 +0700
+++ lcms2-2.19/src/cmslut.c	2026-05-02 14:17:29.729869778 +0700
@@ -460,6 +460,7 @@
 static
 cmsUInt32Number CubeSize(const cmsUInt32Number Dims[], cmsUInt32Number b)
 {
+#ifndef CMS_DONT_USE_INT64
     cmsUInt32Number dim;
     cmsUInt64Number rv;
 
@@ -480,6 +481,28 @@
     if (rv > UINT_MAX / 15) return 0;
 
     return (cmsUInt32Number) rv;
+#else
+    // This is old code from lcms2-2.18
+    cmsUInt32Number rv, dim;
+
+    _cmsAssert(Dims != NULL);
+
+    for (rv = 1; b > 0; b--) {
+
+        dim = Dims[b-1];
+        if (dim <= 1) return 0;  // Error
+
+        rv *= dim;
+
+        // Check for overflow
+        if (rv > UINT_MAX / dim) return 0;
+    }
+
+    // Again, prevent overflow
+    if (rv > UINT_MAX / 15) return 0;
+
+    return rv;
+#endif
 }
 
 static
