From f0cfc211718d9eac519223aa8b3ac7107f437e7e Mon Sep 17 00:00:00 2001
From: Peter Zhigalov <peter.zhigalov@gmail.com>
Date: Sat, 7 Feb 2026 20:52:18 +0700
Subject: [PATCH 1/2] Fix build with TDM-GCC 10.3.0

---
 src/core/others/ojph_mem_c.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/others/ojph_mem_c.c b/src/core/others/ojph_mem_c.c
index 8c66d24..55f8273 100644
--- a/src/core/others/ojph_mem_c.c
+++ b/src/core/others/ojph_mem_c.c
@@ -62,7 +62,7 @@
 #endif
 
 ////////////////////////////////////////////////////////////////////////////////
-#ifdef OJPH_OS_WINDOWS
+#if (defined OJPH_OS_WINDOWS) && (defined _MSC_VER)
   OJPH_EXPORT void* ojph_aligned_malloc(size_t alignment, size_t size)
   {
     assert(alignment != 0 && (alignment & (alignment - 1)) == 0);
-- 
2.54.0

